tests/text_checkers/ai/imatrics_test.py

```diff
@@ -0,0 +1,136 @@
+                                 "interesting tags.</p>",
+                    "headline": "test imatrics",
+                    "slugline": "test imatrics",
+                }
+            ])
+
+    @responses.activate
+    def test_checker(self):
+        """Check that spellchecking is working"""
+        doc = {
+            "service": "imatrics",
+            "item_id": "test_1",
+        }
+        ai_service = get_resource_service('ai')
+        api_url = urljoin(TEST_BASE_URL, "article/concept")
+        responses.add(
+            responses.POST, api_url,
+            json=[
+                {
+                    "weight": 1,
+                    "geometry": {},
+                    "links": [],
+                    "title": "IT",
+                    "type": "topic",
+                    "uuid": "e3c482c0-08a4-3b31-a7f1-e231f1ddffc4"
+                },
+                {
+                    "weight": 0.8387794287831216,
+                    "geometry": {},
+                    "links": [],
+                    "title": "Service",
+                    "type": "topic",
+                    "uuid": "44f52663-52f9-3836-ac45-ae862fe945a3"
+                },
+                {
+                    "weight": 1,
+                    "geometry": {},
+                    "links": [
+                        {
+                            "relationType": "",
+                            "id": "medtop:20000763",
+                            "source": "IPTC",
+                            "uri": "",
+                            "url": ""
+                        }
+                    ],
+                    "title": "informasjons- og kommunikasjonsteknologi",
+                    "type": "category",
+                    "uuid": "c8a83204-29e0-3a7f-9a0e-51e76d885f7f"
+                }
+            ]
+
+        )
+        ai_service.create([doc])
+
+        expected = {
+            "category": [
+                {
+                    "media_topics": [
+                        {
+                            "name": "informasjons- og kommunikasjonsteknologi",
+                            "qcode": "20000763"
+                        }
+                    ],
+                    "title": "informasjons- og kommunikasjonsteknologi",
+                    "uuid": "c8a83204-29e0-3a7f-9a0e-51e76d885f7f",
+                    "weight": 1
+                }
+            ],
+            "topic": [
+                {
+                    "media_topics": [],
+                    "title": "IT",
+                    "uuid": "e3c482c0-08a4-3b31-a7f1-e231f1ddffc4",
+                    "weight": 1
+                },
+                {
+                    "media_topics": [],
+                    "title": "Service",
+                    "uuid": "44f52663-52f9-3836-ac45-ae862fe945a3",
+                    "weight": 0.8387794287831216
+                }
+            ]
+        }
```

Comment:
I'm not sure if we should expose all that api data to clients, imo would be good to convert it to some internal format so we can have generic ui widget, eg I wouldn't send weight but we could just use it to sort on backend, also I would rather stick with ninjs specs so using `name` and `code` and `subject/object/event/person/place` on top level, not sure if this api can return anything but subjects though

Reply:
yeah I was not sure about this one, it's already sorted like with it in the backend, so I guess I can just get rid of it.
Yes this API can return locations, persons, etc.

Reply:
@petrjasek I've put a mapping, but the screenshots at https://wiki.sourcefabric.org/display/NR/Auto-tagging+tool shows iMatrics terms, so I'm not sure if this what the customer expects.
Please check the mapping, how would you map "category"?

Reply:
good question, I suppose it's some kind of subject but not sure

Reply:
otherwise the mapping looks good, I just think we have to also update item schema so it can hold references to places/persons etc otherwise the widget won't be able to make use of it

Reply:
`place` was already there, I've added `person` in latest update.

Reply:
also I'm trying to separate first-name from last-name, but it's a best guess has there is no information in the documentation I have on how the person in formatted beside the `Name LastName` name of the type.